lib/routes/linkedin/jobs.ts

```diff
@@ -1,16 +1,36 @@
 import { Route } from '@/types';
 import got from '@/utils/got';
 import { parseDate } from '@/utils/parse-date';
-import { parseJobSearch, KEYWORDS_QUERY_KEY, JOB_TYPES, JOB_TYPES_QUERY_KEY, EXP_LEVELS_QUERY_KEY, parseParamsToSearchParams, EXP_LEVELS, parseParamsToString } from './utils';
+import {
+    parseJobSearch,
+    KEYWORDS_QUERY_KEY,
+    JOB_TYPES,
+    JOB_TYPES_QUERY_KEY,
+    EXP_LEVELS_QUERY_KEY,
+    parseParamsToSearchParams,
+    EXP_LEVELS,
+    parseParamsToString,
+    GEO_ID_QUERY_KEY,
+    WORK_TYPE,
+    WORK_TYPE_QUERY_KEY,
+    TIME_POSTED_RANGE_QUERY_KEY,
+} from './utils';
 
 const BASE_URL = 'https://www.linkedin.com/';
 const JOB_SEARCH_PATH = '/jobs-guest/jobs/api/seeMoreJobPostings/search';
 
 export const route: Route = {
-    path: '/jobs/:job_types/:exp_levels/:keywords?',
-    categories: ['other'],
+    path: '/jobs/:job_types/:exp_levels/:keywords?/:work_types?/:geo_id?/:time_posted_range?',
```

Comment:
This adds way too many optional parameters. Please use a single optional parameter which accepts them [in URL search string style](https://docs.rsshub.app/routes/social-media#twitter) instead.

Reply:
modified to `path: '/jobs/:job_types/:exp_levels/:keywords?/:routeParams?',`